src/router/bundle/bundleHandler.ts

```diff
@@ -58,8 +58,8 @@ export default class BundleHandler implements BundleHandlerInterface {
         this.validators = validators;
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    async processBatch(bundleRequestJson: any, userIdentity: KeyValueMap, requestContext: RequestContext) {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars,prettier/prettier

```

Comment:
what are we disabling here?

Reply:
Now that we added an extra param, prettier wants the parameters to be one per line but that would require adding an extra `@typescript-eslint/no-unused-vars` on each line.

IMO It looks better to just disable prettier in this case.